pinery-miso/src/main/java/ca/on/oicr/pinery/lims/miso/MisoClient.java

```diff
@@ -95,8 +95,6 @@
   // RunSample queries
   private static final String QUERY_ALL_RUN_SAMPLES = getResourceAsString("queryAllRunSamples.sql");
   private static final String QUERY_RUN_SAMPLES_BY_RUN_ID = QUERY_ALL_RUN_SAMPLES
-      + " JOIN SequencerPartitionContainer_Partition spcp ON spcp.partitions_partitionId = part.partitionId"
-      + " JOIN SequencerPartitionContainer spc ON spc.containerId = spcp.container_containerId"
       + " JOIN Run_SequencerPartitionContainer rcpc ON rcpc.containers_containerId = spc.containerId" + " WHERE rcpc.Run_runId = ?";
```

Comment:
Will deleting (old) line 99 (joining `spc`) cause (new) line 98 to fail when it tries to join on `spc.containerId`?

Reply:
No. It should be fine because it's the query in the separate file now.